src/components/City/EditCity.vue

```diff
@@ -144,6 +144,7 @@ export default {
 			this.editedCountry = this.options[this.arrowCounter].country;
 			this.addCity = false;
 			this.arrowCounter = -1;
+			this.options = [];
 			this.$emit('change', {
```

Comment:
I see this is the only change in the PR. Does not look like it will fix the issue that in the user's profile page the user should be able to select a city by pressing an Enter key. Can you please post a video of the fix? Or join our slack channel to discuss? https://join.slack.com/t/shoonya-tech/shared_invite/zt-i783y3rk-5I97wQmTZ~4zchNZdKJs2w